stage1/init/init.go

```diff
@@ -369,6 +551,14 @@ func stage1() int {
 		return 3
 	}
 
+	appHashes := p.GetAppHashes()
+	s1Root := common.Stage1RootfsPath(p.Root)
+	machineID := p.GetMachineID()
+	if err := createCgroups(s1Root, machineID, appHashes); err != nil {
```

Comment:
Does it require systemd-v220 in stage1? How does it behave when rkt is compiled with `RKT_STAGE1_USR_FROM=coreos` (systemd-v215)? If it fails, maybe the app-isolators feature could be disabled when using `flavor=coreos`?

At the moment, `RKT_STAGE1_USR_FROM=src` still build systemd-v219 and I don't know if we will switch to v220 for the next rkt release. Maybe your two commits could be added in stage1/rootfs/usr_from_src/patches/v219:

http://cgit.freedesktop.org/systemd/systemd/commit/?id=54b4755f15438c86991d5a4eaadc47150f7e5a84
http://cgit.freedesktop.org/systemd/systemd/commit/?id=875e1014dd9d55cd0692dcce843598cffb2d09b0